blog/2025/devnet-reset.md

```diff
@@ -0,0 +1,57 @@
+---
+category: 2025
+date: 2025-01-24
+seo:
+    title: Upcoming Devnet Reset Notification
```

Comment:
We should be able to leave the `title` field off and have it pick the title from the first h1 of the page, I think.

If we can't, we should make sure they match. Right now this doesn't match because it has the extra word "Notification"